### ISSUE TITLE
js_autofocus doesn’t work

### ISSUE THREAD

**Issue by ausi** [contao/core-bundle#1058]
The code from the js_autofocus template https://github.com/contao/core-bundle/blob/60970bc1027a9b293279342ec0158df2b471da73/src/Resources/contao/templates/js/js_autofocus.html5#L7 doesn’t work because `offsetTop` is the offset to the parent element, not the offset to the document. I think we should use something like this instead:

```js
window.scrollTo(null, parseInt(p.getBoundingClientRect().top + window.pageYOffset - 40));
```

**Comment by leofeyer** [comment]
As discussed with @ausi, we'll go for `p.scrollIntoView()` as he suggested [here](https://github.com/contao/core/issues/8044#issuecomment-248291971).

**Comment by leofeyer** [comment]
Fixed in 588f3358acec9173ca829679553b0ad235411384.
